lib/video/postprocessing/finetune/getTimingMetrics.js

```diff
@@ -24,17 +25,20 @@ function get(metric, metricName, pos, options) {
     x = 10;
   }
 
+  // Adjust the range by 1ms to ensure it displays on the correct frames.
+  let range = Number.isFinite(end)
+    ? `between(t,${(Number(start) - 1) / 1000},${(Number(end) - 1) / 1000})`
+    : `gte(t,${(Number(start) - 1) / 1000})`;
   // H-h/8
-  return `drawtext=${fontFile}enable='between(t,${
-    Number(metric) / 1000
-  },30)':x=(w-tw)/2: y=H-${pos}-h/${x}:fontcolor=white:fontsize=h/${fontSize}:box=1:boxcolor=0x000000AA:boxborderw=2:text='${metricName} ${metric}'`;
+  return `drawtext=${fontFile}enable='${range}':x=(w-tw)/2: y=H-${pos}-h/${x}:fontcolor=white:fontsize=h/${fontSize}:box=1:boxcolor=0x000000AA:boxborderw=2:text='${metricName} ${start}'`;
```

Comment:
Do you need to display it on the video or is just getting the metrics in the JSON enough? The reason I ask is that the more changes done in the original code, the more I need to test and verify that old functionality still works.

Reply:
It would be nice to put in the review, but I removed it to make your review simpler. Maybe I can provide it in a future PR of a lower priority :).